python-checks/src/main/resources/org/sonar/l10n/py/rules/python/S1854.html

```diff
@@ -0,0 +1,22 @@
+<p>A dead store happens when a local variable is assigned a value that is not read by any subsequent instruction. Calculating or retrieving a value
+only to then overwrite it or throw it away, could indicate a serious error in the code. Even if it's not an error, it is at best a waste of resources.
+Therefore all calculated values should be used.</p>
+<h2>Noncompliant Code Example</h2>
+<pre>
+i = a + b // Noncompliant; calculation result not used before value is overwritten
+i = compute()
+</pre>
+<h2>Compliant Solution</h2>
+<pre>
+i = a + b
+i += compute()
+</pre>
+<h2>Exceptions</h2>
+<p>This rule ignores initializations to -1, 0, 1, <code>null</code>, <code>true</code>, <code>false</code> and <code>""</code>.</p>
```

Comment:
We should change this exception description by replacing `null, true, false` by `None, True, False` and implement it :stuck_out_tongue_winking_eye:.
It will remove this kind of somewhat false-positive: [twisted/words/protocols/jabber/jid.py#L34](https://github.com/SonarSource/python-test-sources/blob/aa34d4afeb8ee40b6750d9d2b44ce98a78043771/twisted-12.1.0/twisted/words/protocols/jabber/jid.py#L34)